File: AngularBooking/ClientApp/src/app/core/_models/entity/room.ts
import { Showing } from "./showing";

export class Room{
  constructor(
    public id?: number,
    public venueId?: number,
    public name?: string,
    public description?: string,
    public rows?: number,
    public columns?: number,
    public isles?: string,
    public showings?: Showing[]
  ) { }
}

File: AngularBooking/Models/Room.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AngularBooking.Models
{
    public class Room : IModel
    {
        public int Id { get; set; }
        public int VenueId { get; set; }
        public string Name { get; set; }
        public string Description { get;set; }
        public int Rows { get; set; }
        public int Columns { get; set; }
        public string Isles { get; set; } // JSON

        // navigation
        public Venue Venue { get; set; }
        public List<Showing> Showings { get; set; }
    }
}
